package.json

```diff
@@ -24,7 +24,7 @@
     "build:publish": "npm run build && npm run lint && npm run check && npm run doc && npm run test:retry",
     "postbuild:publish": "script/postbuild",
     "predoc": "rm -rf docs && node script/build-guides.js",
-    "doc": "node node_modules/typedoc/bin/typedoc --options typedoc.json --gitRemote https://github.com/aws/amazon-chime-sdk-js",
+    "doc": "script/generate-docs",
```

Comment:
So when you run npm run build:release from a fork, will it use upstream/master?

Reply:
Correct. Locally:

```
> script/generate-docs

Generating docs against upstream/master.
```

and [in Travis it uses origin/master](https://travis-ci.org/github/aws/amazon-chime-sdk-js/jobs/730261169#L268).